src/probnum/problems/zoo/diffeq/_ivp_examples_jax.py

```diff
@@ -0,0 +1,168 @@
+import numpy as np
+
+from probnum.problems import InitialValueProblem
+
+__all__ = ["threebody_jax", "vanderpol_jax"]
+# pylint: disable=import-outside-toplevel
+
+
+JAX_ERRORMSG = "IVP instantiation requires jax. Try using `threebody()` instead, or install `jax` via `pip install jax jaxlib`"
+
+
+def threebody_jax(tmax=17.0652165601579625588917206249):
```

Comment:
this is not consistent with the formulation in diffeq.ivp_examples, but I prefer it like this and will update the other variant soon.